custom_components/detailed_hello_world/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Detailed Hello World",
```

Comment:
Title is only needed for internal manifests. HA will use the manifest title by default, which is fine. 
```suggestion
```

Reply:
Done.
This was created by the scaffold. When I get to updating as per comment above, I'll also remove these title values if you agree?

Reply:
Yes, sounds good.